core/tests/protractor_utils/workflow.js

```diff
@@ -18,13 +18,27 @@
  */
 
 var forms = require('./forms.js');
+var path = require('path');
 var users = require('./users.js');
 var waitFor = require('./waitFor.js');
 var CreatorDashboardPage = require('./CreatorDashboardPage.js');
 var ExplorationEditorPage = require('./ExplorationEditorPage.js');
 var LibraryPage = require('./LibraryPage.js');
 var TopicsAndSkillsDashboardPage = require('./TopicsAndSkillsDashboardPage.js');
 
+//
```

Comment:
Why the //, here and below? (Please proofread your PR.)

Reply:
Done.